contrib/babelfishpg_tsql/src/pltsql.h

```diff
@@ -2348,4 +2348,29 @@ extern bool validate_special_function(char *proc_nsname, char *proc_name, int na
  */
 extern char *tsql_format_type_extended(Oid type_oid, int32 typemod, bits16 flags);
 
+/* Restricted objects definition */
+typedef struct
+{
+	const char *name;
+	ObjectType type;
+} RestrictedObject;
+
+/* Separate arrays for each schema */
+static const RestrictedObject master_dbo_objects[] = {
+	{"xp_qv",					OBJECT_PROCEDURE},
+	{"xp_instance_regread",		OBJECT_PROCEDURE},
+	{"sp_addlinkedserver",		OBJECT_PROCEDURE},
+	{"sp_addlinkedsrvlogin",	OBJECT_PROCEDURE},
+	{"sp_droplinkedsrvlogin",	OBJECT_PROCEDURE},
+	{"sp_dropserver",			OBJECT_PROCEDURE},
+	{"sp_enum_oledb_providers",	OBJECT_PROCEDURE},
+	{"sp_testlinkedserver",		OBJECT_PROCEDURE}
+};
+
+static const RestrictedObject msdb_dbo_objects[] = {
+	{"syspolicy_configuration",				OBJECT_VIEW},
+	{"syspolicy_system_health_state",		OBJECT_VIEW},
+	{"fn_syspolicy_is_automation_enabled",	OBJECT_FUNCTION}
+};
```

Comment:
I think we are already maintaining these names inside [is_ms_shipped](https://github.com/babelfish-for-postgresql/babelfish_extensions/blame/BABEL_5_X_DEV/contrib/babelfishpg_tsql/runtime/functions.c#L3838) function, which can be declare globally and used here as well.